TeamZeroServer/database.sql

```diff
@@ -0,0 +1,46 @@
+CREATE TABLE users(
+    user_id serial PRIMARY KEY,
+    username VARCHAR (50) UNIQUE NOT NULL,
+    email VARCHAR (355) UNIQUE NOT NULL,
+    password VARCHAR (50) NOT NULL
+);
+
+CREATE TABLE user_info(
+    user_id integer PRIMARY KEY,
+    avatar BYTEA,
+    FOREIGN KEY (user_id) REFERENCES users (user_id)
+);
+
+CREATE TABLE groups(
+    group_id serial PRIMARY KEY,
+    group_name VARCHAR (50) UNIQUE NOT NULL,
+    number_of_members INTEGER NOT NULL,
+    group_avatar BYTEA
+);
+
+CREATE TABLE user_groups(
+    user_id integer,
+    group_id integer,
+    FOREIGN KEY (user_id) REFERENCES users (user_id),
+    FOREIGN KEY (group_id) REFERENCES groups (group_id)
+);
```

Comment:
So this will be a one to many relationship right? one user_id might be associated with many group_ids?

is there a way to specify this relationship, or will it just be determined in how we use the user and group tables in the database?